js/packages/core/src/ufg/types.ts

```diff
@@ -46,7 +52,7 @@ export interface Eyes<TSpec extends SpecType> extends AutomationCore.Eyes<TSpec>
 }
 
 export type CheckSettings<TSpec extends SpecType> = AutomationCore.CheckSettings<TSpec> & {
-  renderers?: Renderer[]
+  renderers?: RendererWithProps[]
```

Comment:
Then there is no need in this type anymore

```suggestion
renderers?: (Renderer & {properties?: BaseCore.CustomProperty})[]
```